specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -728,25 +736,27 @@
 					"$ref": "./examples/SearchServiceCreateSkillset.json"
 				}
 			},
-			"description": "Creates a new Cognitive Search skillset.",
+			"description": "Creates a new Cognitive Search skillset under the search service.",
```

Comment:
under the search service -> in an Azure Search service.

Reply:
fixed